apps/minifront/src/abort-loader.ts

```diff
@@ -32,8 +32,12 @@ const retry = async (fn: () => boolean, ms = 500, rate = Math.max(ms / 10, 50))
  * timeout. This is a temporary solution until loaders properly await Prax
  * connection.
  */
-export const abortLoader = async (): Promise<void> => {
+export const abortLoader = async (): Promise<null> => {
```

Comment:
Is this the trick, ie. returning `null`, that enables instantaneous route changes?